@here/harp-mapview/lib/ThemeLoader.ts

```diff
@@ -113,18 +127,17 @@ export class ThemeLoader {
         if (typeof theme === "string") {
             const uriResolver = options.uriResolver;
             const themeUrl = uriResolver !== undefined ? uriResolver.resolveUri(theme) : theme;
-
             const response = await fetch(themeUrl, { signal: options.signal });
             if (!response.ok) {
                 throw new Error(`ThemeLoader#load: cannot load theme: ${response.statusText}`);
             }
             theme = (await response.json()) as Theme;
             theme.url = resolveReferenceUri(getAppBaseUrl(), themeUrl);
-            theme = this.resolveUrls(theme, uriResolver);
+            theme = this.resolveUrls(theme, uriResolver, options);
```

Comment:
Minor, we can pass just `options` as `uriResolver` is already in `options`.

Reply:
I didn't want to change this method b/c it is public API, but I think it should not be public so I will make it private

Reply:
done. I made all resolve methods private. They were only used in tests and for custom theme loading in mapview-demo.